src/tests/unit/tests/electron/platform/android/setup/live-android-setup-deps.test.ts

```diff
@@ -22,6 +23,9 @@ describe('LiveAndroidSetupDeps', () => {
     let configMessageCreatorMock: IMock<UserConfigMessageCreator>;
     let fetchConfigMock: IMock<DeviceConfigFetcher>;
     let loggerMock: IMock<Logger>;
+    let adbWrapperFactoryMock: IMock<AdbWrapperFactory>;
+    let adbWrapperHolderMock: IMock<AdbWrapperHolder>;
+    let adbWrapperStub: AdbWrapper;
```

Comment:
My inclination would be to use a strict mock for adbWrapperStub, but that may just be my C# reflexes kicking in...

Reply:
Since there is method invocation happening or a snapshot being taken for the wrapper, we only care about the identity (when interacting with other mocks to validate it's being passed through), so it's not needed here.